recipes/lrzip/meta.yaml

```
@@ -10,7 +10,7 @@ source:
   sha256: 48bd8decb097c1596c9b3777959cd3e332819434ed77a2823e65aa436f1602f9
 
 build:
-  number: 0
+  number: 1
   skip: True # [osx]
```

```suggestion
  skip: True # [osx and x86_64]
```
Right now it's skipping all osx. If you want to exclude osx-64 but build osx-arm64, you can make this change. Or if you want both, just remove this skip line.

---

Oh thanks for noticing that @aliciaaevans! I was bumping this because I need osx-arm64 lrzip for some of our other tools to build on osx-arm64. Not sure why it was set to skip osx before - will try removing that!